src/pages/transaction/nft/transfer.vue

```diff
@@ -1,12 +1,22 @@
 <template>
   <div class="transactionPage transferNFTPage dappPageWrapper">
-    <block-transaction ref="transactionBlock" />
+    <block-transaction :fromRoute="fromRoute" ref="transactionBlock" />
   </div>
 </template>
 
 <script lang="ts">
 import Vue from "vue";
 export default Vue.extend({
+  asyncData({ from }) {
```

Comment:
And this also can be moved to some kind of middleware (e.g. [routerMiddleware](https://nuxtjs.org/docs/directory-structure/middleware#router-middleware))